crates/cdk-mint-rpc/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "cdk-mint-rpc"
+version = "0.1.0"
```

Comment:
All other CDK crates have a "unified", single version. Since there are so many inter-dependencies, maybe it's good to stick with that model and use the CDK version here (0.6.0)?